udata_front/theme/gouvfr/templates/header.html

```diff
@@ -78,15 +81,15 @@
         <nav class="header-nav">
             <div class="hagrid">
                 <div class="row justify-between align-items-center">
-                    <nav class="nav-tabs col">
+                    <nav class="nav-tabs col" role="navigation">
```

Comment:
Not sure if this is an issue, but on mobile, the navigation role appears twice.

Reply:
We can have multiple navigations per page but we may want to add an aria-label to them (as stated here https://disic.github.io/guide-integrateur/1-gabarit-general.html#html5aria)

Reply:
ah yes indeed, nice ! :+1: